src/config/networks/polygon.ts

```diff
@@ -25,7 +25,9 @@ const polygon: NetworkConfig = {
     },
     staging: {
       ...baseConfig,
-      safeUrl: 'https://safe-team-polygon.staging.gnosisdev.com/app/',
```

Comment:
I assume `safeUrl` is removed because we'll have the same URL for all networks?

Reply:
Added it back: 41bef84